Automattic/vip-go-mu-plugins
# Update Jetpack 9.3 to 9.3.2

## Description

Update `jetpack-9.3` subtree to version 9.3.2

## Changelog Description

### Plugin Updated: Jetpack 9.3.2

We upgraded Jetpack 9.3 to the latest point release 9.3.2.

There's a single change which enhaces the sync process to avoid concurrent operations, which is something that will potentially benefit some big VIP sites:

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Go to the Jetpack dashboard in wp-admin
1. Ensure version 9.3.2 is active
